src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -517,6 +517,23 @@ public T WithTelemetry(ITelemetryConfig telemetryConfig)
             return (T)this;
         }
 
+        /// <summary>
+        /// Sets telemetry client for the application.
```

Comment:
Any documentation that describes how to use this? Or what it is for? Or is this for internal use only?

Reply:
I will add documentation here after I create it. This api is for first party only